### DISCUSSION TITLE
How to automatically connect to Server when loading project??

### DISCUSSION THREAD

**Discussion by samallen101** [apohl79/audiogridder#582] [Q&A]
Hey there, every time I open up a session in my DAW, I have to go through each individual instance of AudioGridder and connect to server.

Is there a way to skip, so that it automatically connects?

**Comment by CaBuffalo** [comment]
Yes, it connects automatically w/o any extra steps. If it doesn't work for
some reason, you can open the APP tray and have your DAW connect to the AG
server by just one click.

On Sat, May 22, 2021 at 9:24 PM samallen101 ***@***.***>
wrote:

> Hey there, every time I open up a session in my DAW, I have to go through
> each individual instance of AudioGridder and connect to server.
>
> Is there a way to skip, so that it automatically connects?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/582>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GHZGIE5TIZZYAV4CLTPB7QDANCNFSM45LJQP3A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
> Hey there, every time I open up a session in my DAW, I have to go through each individual instance of AudioGridder and connect to server.
> 
> 
> 
> Is there a way to skip, so that it automatically connects?

sounds like your server IP changes. AG plugins reconnect to the last IP.
